edgelet/iotedged/src/lib.rs

```diff
@@ -161,15 +161,19 @@ enum StartApiReturnStatus {
 
 pub struct Main {
     settings: Settings<DockerConfig>,
+    hsm_lock: Arc<HsmLock>,
```

Comment:
This is only used inside `Main::run`, so just instantiate it in the function instead of making it a field of `Main`.

Reply:
Done.